spec/unit/block_score/connection_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'faker'
+
+RSpec.describe BlockScore::Connection do
+  before do
+    BlockScore.api_key = ENV.fetch(BlockScore::ENVIRONMENT_API_KEY,
```

Comment:
This is setting a global variable in a before block, but it doesn't reset the variable back to the original value afterwards. Are there other tests that might depend on the value of this global variable?